src/app/login/login-form.component.ts

```diff
@@ -49,21 +80,17 @@ export class LoginFormComponent {
     this.router.navigate([ this.returnUrl ]);
   }
 
-  createUser({ name, password, repeatPassword }: {name: string, password: string, repeatPassword: string}) {
-    if (password === repeatPassword) {
+  createUser(name: string, password: string, repeatPassword: string) {

```

Comment:
This does not need to change.  Previous version was using object destructuring (which you can read up about in the [Typescript docs](https://www.typescriptlang.org/docs/handbook/variable-declarations.html)) allowing us to simply pass the object to the function rather than each property as a separate argument.